src/compose.js

```diff
@@ -6,6 +6,8 @@ var reverse = require('./reverse');
  * Performs right-to-left function composition. The rightmost function may have
  * any arity; the remaining functions must be unary.
  *
+ * **Note:** the result of compose is not automatically curried.
```

Comment:
Could we also `s/the/The/`? I recall reading that if there's just one word (or perhaps just a few words) before the colon, the word after the colon should be capitalized.